style.css

```diff
@@ -912,12 +921,14 @@ input:focus:invalid {
   }
   #contact-form {
     position: relative;
-    padding: 10px;
+    padding: 6px;
+    width: 60%;
+    margin-right: 3rem;
   }
   
 .input2{
     display: flex;
-    position: absolute;
+     position: absolute;
```

Comment:
- [ ] As requested by the previous reviewer I would suggest you please stick to the design as much as possible. Please adjust the spacing by looking at the design below.
 [Optional] Please use the same color for the content so it looks like the original design.

- [ ] Your Design 
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/68823331/2cdd3384-70e1-4d83-a745-269f9c3c0337)


 - [ ] Figma Design
 
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/68823331/c971121c-8523-45db-9b49-7e8b83608d78)